Items/T3/Scepter.cs

```diff
@@ -260,7 +260,8 @@ private class ScepterReplacer {
         }
 
         private void Reroll(CharacterBody self, int count) {
-            if(count <= 0) return;
+            var bodyName = BodyCatalog.GetBodyName(self.bodyIndex);
```

Comment:
I think it's better this part of code to respect the config related to scepter. Please update the code accordingly.

Reply:
I'm unsure of what you mean. The config option for rerolling extra scepters is checked at line 256. The Reroll method is then called if the option is set to true. The proposed code change does not interfere with it.

Reply:
My memory is unclear, and I'm unsure now. I do remember that turrets will still get a reroll if the config option is enabled or not. So maybe it's a bug somewhere else.

Reply:
If the config option to reroll extra scepters is enabled, when an Engineer picks up an extra scepter, they will receive a random red item, and the turrets will inherit that red item.

The current behavior happens because the Engineer can benefit from having a copy of the Scepter, and the engineer's turrets, which inherit the Scepter, do not benefit. It gets rerolled because it is "unusable" for them, using the edited function.

There are also config options that reduce the cooldown for placing turrets if you have a Scepter. This code change does not affect those options.